packages/manager/src/views/WebPropertyDetailPage/components/workflow3.0/ConfigureWorkflowForm.tsx

```diff
@@ -86,21 +89,26 @@ const schema = yup.object({
 interface Props {
   onClose: () => void;
   propertyIdentifier: string;
-
   dataProps: TDataWorkflow | TDataContainerized;
   flag: string;
 }
 type MapItem = {
   name: string;
   value: string;
 };
+
 const keyValuePairsGenerator = ({
   dataProps,
   property
 }: {
-  dataProps: TDataWorkflow | TDataContainerized;
+  dataProps: Record<string, any>;
   property: string;
-}) => Object.entries(dataProps?.[property] || {}).map(([key, value]) => ({ key, value }));
+}) =>
+  Object.entries(dataProps?.[property] || {}).map(([key, value]) => ({
+    key,
+    value: Base64.decode(value as string), // Type assertion to enforce string type
+    isSecret: property === 'spashipWorkflowSecret'
```

Comment:
`spashipWorkflowSecret` use in a variable as it's being used multiple places

Reply:
Fixed